src/modules/experts/store/expertsSlice.ts

```diff
@@ -56,7 +56,7 @@ const initialState: IExpertsState = {
   experts: {
     experts: [],
     meta: {
-      totalPages: 0,
+      totalPages: undefined,
```

Comment:
is it supposed to be `undefined`?

Reply:
> is it supposed to be `undefined`?

Yes, since on the client-side we don't know the starter amount of pages. So we set them after a successful response from the backend.

Reply:
looks strange but maybe makes sense